src/topp/SageAdapter.cpp

```diff
@@ -473,16 +1121,27 @@ class TOPPSageAdapter :
     arguments << config_file.toQString() 
               << "-f" << fasta_file.toQString() 
               << "-o" << output_folder.toQString() 
-              << "--write-pin";
+              << "--annotate-matches"
+              << "--write-pin"; 
+
     if (batch >= 1) arguments << "--batch-size" << QString(batch);
     for (auto s : input_files) arguments << s.toQString();
 
     OPENMS_LOG_INFO << "Sage command line: " << sage_executable << " " << arguments.join(' ').toStdString() << std::endl;
+    
+    //std::chrono lines for testing/writing purposes only! 
```

Comment:
remove commented out code or put in a macro #idef that can be enabled / disabled

Reply:
Alright, I'll put it in a macro then!